src/main/java/cz/czechitas/automation/ApplicationDetail.java

```diff
@@ -19,10 +19,11 @@ final class ApplicationDetail {
     }
 
     void vyberTermin(String termin) {
-        var dateElement = elementFinder.findByXPath("/html/body/div/div/div/div/div/form/table/tbody/tr[2]/td[2]/div/button");
+        var dateElement = elementFinder.findByXPath("//button[@data-id='term_id']");
         dateElement.click();
-        //TODO: find by parameter instead of selecting first option
-        var option = elementFinder.findByXPath("//*[@id=\"bs-select-1-0\"]");
+        var option = elementFinder.findByXPath(
+                "//div[starts-with(@id,'bs-select')]//span[contains(text(), '" + termin + "')]"
```

Comment:
There will be probably conflict in the naming after the translation to the english.